### PR TITLE
refactor sheet structure

### PR DESCRIPTION
**Pull request by akash1810**
- moving from one tab per documentary to a single tab with multiple rows to stop breaking the visuals tool
- new sheet - https://docs.google.com/spreadsheets/d/1jYb4okCk7I42wCoduH-o48DE-EKGajM8w-MTxQYxmck/edit#gid=1344855923

# Sheet JSON before
```json
{
  "sheets": {
    "coming-soon": [
      {
        "title": "Fish Story",
        "strapline": "A search for the truth behind a fishy tale.",
        "date": "Out 30 June",
        "image": "https://media.guim.co.uk/d69765463f6abc4a725c15599ebe77e46f9df262/353_326_1280_427/1280.jpg"
      },
      {
        "title": "Pitching Up",
        "strapline": "The Irish town reaching refugees to hurl",
        "date": "Out 14 July",
        "image": "https://media.guim.co.uk/b8bdf607d8568f275165c365ec4a849358e54299/640_89_1280_427/1280.jpg"
      }
    ],
    "gun-nation": [
      {
        "title": "Gun Nation",
        "shortDescription": "A revealing and unsettling journey to the heart of America’s deadly love affair with the gun",
        ...
      }
    ],
    "gun-nation-chapters": [
      {
        "chapterTitle": "Intoduction",
        "Put chapter minutes here": "0",
        "Seconds here": "0",
        "chapterTimestamp": "0"
      }, ...
    ],
    "desert-fire": [
      {
        "title": "Desert Fire",
        "shortDescription": "Football’s world cup for unrecognised territories is viewed through the eyes of the manager and players of one of the most fascinating teams, Iraqi Kurdistan",
        ...
      }
    ],
    "desert-fire-chapters": [
      {
        "chapterTitle": "Introduction",
        "Put chapter minutes here": "0",
        "Seconds here": "0",
        "chapterTimestamp": "0"
      },
      ...
    ]
  }
}
```

# Sheet JSON after
```json
{
  "sheets": {
    "coming-soon": [
      {
        "title": "Fish Story",
        "strapline": "A search for the truth behind a fishy tale.",
        "date": "Out 30 June",
        "image": "https://media.guim.co.uk/d69765463f6abc4a725c15599ebe77e46f9df262/353_326_1280_427/1280.jpg"
      },
      {
        "title": "Pitching Up",
        "strapline": "The Irish town reaching refugees to hurl",
        "date": "Out 14 July",
        "image": "https://media.guim.co.uk/b8bdf607d8568f275165c365ec4a849358e54299/640_89_1280_427/1280.jpg"
      }
    ],
    "documentaries": [
      {
        "docName": "gun-nation",
        "title": "Gun Nation",
        "shortDescription": "A revealing and unsettling journey to the heart of America’s deadly love affair with the gun",
        ...
      },
      {
        "docName": "desert-fire",
        "title": "Desert Fire",
        "shortDescription": "Football’s world cup for unrecognised territories is viewed through the eyes of the manager and players of one of the most fascinating teams, Iraqi Kurdistan",
        ...
      }
    ],
    "chapters": [
      {
        "docName": "gun-nation",
        "documentary": "Gun Nation",
        "chapterTitle": "Intoduction",
        "minutes": "0",
        "seconds": "0",
        "chapterTimestamp": "0"
      },
      {
        "docName": "gun-nation",
        "documentary": "Gun Nation",
        "chapterTitle": "The gun owners",
        "minutes": "1",
        "seconds": "52",
        "chapterTimestamp": "112"
      },
      ...,
      {
        "docName": "desert-fire",
        "documentary": "Desert Fire",
        "chapterTitle": "Introduction",
        "minutes": "0",
        "seconds": "0",
        "chapterTimestamp": "0"
      },
      {
        "docName": "desert-fire",
        "documentary": "Desert Fire",
        "chapterTitle": "The coach, Khasraw",
        "minutes": "1",
        "seconds": "0",
        "chapterTimestamp": "60"
      },
      ...
    ]
  }
}
```

TODO
- [x] migrate all doc metadata to new sheet
- [x] demo to Editorial

### CODE REVIEW

**Comment by akash1810** [comment]
cc @fcage

File: src/js/lib/sheettodom.js [hunk 1]
```diff
@@ -14,16 +14,18 @@ export function sheetToDomInnerHtml({sheetId, sheetName, el, comingSoonSheetName
...
+                    const field = node.getAttribute('data-sheet-attribute');
 
-                    if(startsWith(value, `${comingSoonSheetName}-`)){
-                        node.innerHTML = resp.sheets[comingSoonSheetName][0][value.split(`${comingSoonSheetName}-`)[1]];
+                    if(startsWith(field, `${comingSoonSheetName}-`)){
+                        node.innerHTML = resp.sheets[comingSoonSheetName][0][field.split(`${comingSoonSheetName}-`)[1]];
                     }
                     else {
-                        node.innerHTML = resp.sheets[sheetName][0][value];
+                        const docData = resp.sheets.documentaries.find(_ => _.docName === docName);
```
Comment (gidsg) [inline src/js/lib/sheettodom.js]: Can we throw a `console.warn` if this is `null` to help diagnose authoring issues?

Comment (akash1810) [inline src/js/lib/sheettodom.js]: I'm going to refactor this in an incoming PR, creating a module for doc metadata and DRYing out the code as we're repeating this here - https://github.com/guardian/docs-interactive-template/pull/134/files#diff-691ff2b74e21c3011d7cec25c48f6146R69

Comment (akash1810) [inline src/js/lib/sheettodom.js]: PR - https://github.com/guardian/docs-interactive-template/pull/137

**Pull request merged**